libraries/src/Component/ComponentHelper.php

```diff
@@ -199,15 +199,18 @@ public static function filterText($text)
 					}
 				}
 
-				// Collect the blacklist or whitelist tags and attributes.
+				// Collect the disallowList or allowlist tags and attributes.
```

Comment:
```suggestion
				// Collect the Disallow List or Allow List tags and attributes.
```
Formalise the wording to show as separate words.